src/plugins/search/component.js

```diff
@@ -171,11 +171,6 @@ function bindEvents() {
 
   let timeId;
   // Prevent to Fold sidebar
-  Docsify.dom.on(
-    $search,
-    'click',
-    e => e.target.tagName !== 'A' && e.stopPropagation()
-  );
```

Comment:
can you comment this instead of delete. also add why this is being commented.
use block comments (`/** ... */`)

Reply:
He said here https://github.com/docsifyjs/docsify/pull/1111#discussion_r408640781

Reply:
yeah, this is just for future reference. we can add a link to this PR instead of details.

Reply:
Well, I tested the mobile end and he'll affect the sidebar.

Reply:
can you share the details in a separate review so that he can take a look